locales/en-US/snoozetabs.properties

```diff
@@ -0,0 +1,45 @@
+extDesc=An add-on to let you snooze your tabs for a while.
+popupTitle=Snooze tab until…
+contextMenuTitle=Snooze tab until…
+bookmarkFolderTitle=Snoozed Tabs
+notificationTitle=Tab woke up…
+
+confirmIconAltText=tab icon
+# LOCALIZATION NOTE: placeholder is the time the tab will snooze until, as per https://github.com/bwinton/SnoozeTabs/issues/60
+confirmTimeTitle=This tab will snooze until $TIME$.
+confirmOkButton=OK
+confirmCancelButton=Cancel
+confirmDontShowLabel=Donʼt show this again
+confirmCloseAltText=close button
+
+# LOCALIZATION NOTE: This will be in 5 seconds, for testing.
+timeRealSoonNow=Real Soon Now!
+timeLaterToday=Later Today
+timeTomorrow=Tomorrow
+timeThisWeekend=This Weekend
+timeNextWeek=Next Week
+timeNextMonth=Next Month
+# LOCALIZATION NOTE: This will be the next time Firefox opens.
+timeNextOpen=Next Open
+timeNextOpenLong=the next time Firefox opens
+timePickADate=Pick a Date/Time
+timeUpcomingToday=later today at $TIME$
+timeUpcomingTomorrow=tomorrow at $TIME$
+timeUpcomingLater=$WEEKDAY$, $MONTH$ $DATE$ at $TIME$
```

Comment:
Changed!

Uh, people can't remove stuff yet, but I think I'll need to add that.